packages/launchpad/cypress/e2e/integration/open-mode.spec.ts

```diff
@@ -103,7 +86,7 @@ describe('Launchpad: Open Mode', () => {
 
       cy.visitLaunchpad()
       cy.get('a').contains('Projects').click()
-      cy.findByTestId('project-card')
+      cy.get('[data-cy="project-card"]')
```

Comment:
We can continue to use findByTestId if we configure testing-library:

```js
import { configure } from '@testing-library/cypress'

configure({ testIdAttribute: 'data-cy' })
```

I've made this change in one of my PRs, it might worth a separate PR to make the change and get all the existing tests updated.